### DISCUSSION TITLE
How to navigate back to the previous page after clicking on a link?

### DISCUSSION THREAD

**Discussion by tigerinus** [johnfactotum/foliate#1203] [Q&A]
Apps like `eBoox` on Android has such nice feature - it's really convenient to be able to go back to the previous page after clicking a link in that page.

Is it currently supported in latest version?

Thanks.

**Comment by johnfactotum** [comment]
Yes there are back and forward buttons on the navbar. You can also use <kbd>Alt</kbd> + <kbd>←</kbd> and <kbd>Alt</kbd> + <kbd>→</kbd>.
